sharding/data_modeling.rst

```diff
@@ -10,6 +10,31 @@ Typically, you should choose that column as the distribution column which is the
 .. note::
   Citus also allows joining on non-distribution key columns by dynamically repartitioning the tables for the query. Still, joins on non-distribution keys require shuffling data across the cluster and therefore aren’t as efficient as joins on distribution keys.
 
-The best choice of distribution column varies depending on the use case and queries. Two common scenarios are the multi-tenant B2B application and the realtime analytics dashboard. In the former all tables include a tenant id and are distributed by it. When SQL queries are restricted to accessing data about a single tenant then Citus can execute them within a single shard. Having all data colocated in a shard is efficient and supports all SQL features. However running queries on a single shard limits the ability to parallelize execution.
+Use Cases and Tradeoffs
```

Comment:
Maybe remove this top-level section, and just dive into a discussion of the two separate data models?